lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -432,9 +432,9 @@ def destroy(force_delete: false)
         current_time = Time.current
         target_datetime = valid_datetime || current_time
 
-        with_transaction_returning_status do
```

Comment:
:memo: `with_transaction_returning_status` is not `requires_new: true`, so it may not be Rollback correctly when nested `transaction`